src/NzbDrone.Common/Extensions/StringExtensions.cs

```diff
@@ -131,7 +131,7 @@ public static bool ContainsIgnoreCase(this string text, string contains)
 
         public static string WrapInQuotes(this string text)
         {
-            if (!text.Contains(" "))
+            if (!text.Contains(' '))
```

Comment:
Single quotes feels weird here, even though it's just a char, is there an advantage to it?

Reply:
From Microsoft rule docs:

When searching for a single character, using string.Contains(char) offers better performance than string.Contains(string).